File: Problem1/MultiplesOfThreeAndFive.py
def threeOfFive(max):
    sum = 0
    for x in range(max):
        if x % 3 == 0:
            sum += x
        elif x % 5 == 0:
            sum += x
    return sum

if __name__ == '__main__':
    print(threeOfFive(1000))

File: Problem1/MultiplesOfThreeAndFive.hs
main :: IO()
main = print (sumMultiples 1000)

sumMultiples :: Integral a => a -> a
sumMultiples max = sum (map threeOfFive [0 .. (max - 1)])

threeOfFive :: Integral a => a -> a
threeOfFive num
    | mod num 3 == 0 = num
    | mod num 5 == 0 = num
    | otherwise = 0